src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -2199,16 +2304,6 @@ IsSelectClause(Node *clause)
 	Var *firstColumn = NULL;
 	Index firstColumnTableId = 0;
 	bool isSelectClause = true;
-	NodeTag nodeTag = nodeTag(clause);
```

Comment:
I moved these checks to a separate function given that I don't think it is meaningful to error out for sublinks here. Does that work?